tests/integ/test_huggingface.py

```diff
@@ -111,17 +101,12 @@ def test_huggingface_training(
 
 
 @pytest.mark.release
-@pytest.mark.skipif(
-    integ.test_region() in integ.TRAINING_NO_P2_REGIONS
-    and integ.test_region() in integ.TRAINING_NO_P3_REGIONS,
-    reason="no ml.p2 or ml.p3 instances in this region",
-)
 @pytest.mark.skip(
     reason="need to re enable it later t.corp:V609860141",
 )
```

Comment:
Can you remove `@pytest.mark.skip` as well?

Reply:
done